yarn-project/simulator/src/client/unconstrained_execution.test.ts

```diff
@@ -63,7 +63,7 @@ describe('Unconstrained Execution test suite', () => {
 
       const execRequest: FunctionCall = {
         to: contractAddress,
-        functionData: new FunctionData(FunctionSelector.empty(), true),
+        functionData: new FunctionData(FunctionSelector.empty(), true, false),
```

Comment:
Please add `/*paramName=*/ true` to the true/falses, otherwise we can't know what they are :)

(Ideally follow this convention everywhere else as well)